### DISCUSSION TITLE
BigQuery Destination V2 wants me to upgrade to 0.63.0 but I'm already on 0.63.2

### DISCUSSION THREAD

**Discussion by xavierosee** [airbytehq/airbyte#40606] [Connector Questions]
Hi,

I've recently upgraded to 0.63.2 to migrate to Destinations V2.
The first sync I'm trying to run give me the following error:
`You must upgrade your platform version to use this connector version. Either downgrade your connector or upgrade platform to 0.63.0`

But I migrated to 0.63.2 yesterday and eveything else seems to be working fine.

Connector versions are: 
- BigQuery: 2.7.1
- Amplitude:  0.1.22

Is there something may have done that prevents the connector from running?

**Comment by NAjustin** [comment]
@xavierosee While the source doesn't really have much to do with Destinations V2, that's a _very_ old version of the Amplitude source (currently `0.3.14`), so I would upgrade that to start. It's likely that it's old enough that it's just an incorrect error message. I would try to upgrade to the latest BigQuery destination version (`2.8.1`) to be safe as well.

**Comment by es-habacus** [comment]
Hello, sorry to bump the discussion, but I'm facing same error.  Just updated airbyte to 0.63.5 but still facing the error with BigQuery v2.8.1. What else can I do?

**Reply by NAjustin** [reply]
@es-habacus What version of the Amplitude connector do you have installed?

**Reply by es-habacus** [reply]
I don't have Amplitude connector, problem is related to BigQuery, but found an existing issue: https://github.com/airbytehq/airbyte/issues/39900

[a comment by an automated account is not shown]
